src/implementations/utils/wallet.ts

```diff
@@ -27,7 +27,7 @@ export const getWallet = async ({
   network,
   encryptedWalletPath,
   progress = defaultProgress("Decrypting Wallet"),
-}: WalletOption & NetworkOption & { progress?: (progress: number) => void }): Promise<Wallet> => {
+}: WalletOption & Partial<NetworkOption> & { progress?: (progress: number) => void }): Promise<Wallet> => {
```

Comment:
Why is partial needed? seemed like it's not needed...

Reply:
that's a type is used in many place where it's needed